### DISCUSSION TITLE
Exputt putting in combo with GC2

### DISCUSSION THREAD

**Discussion by dmike2b** [springbok/MLM2PRO-GSPro-Connector#91] [General]
Is it possible to create the exputt putting functionality in combination with the GSPro Simple Control connector (Simple COM port interface) that I am using with my Foresight Sports GC2? The quality of GC2 readings on puts, in my opinion, is not that great. It would be awesome if we could use the Exputt for putting combined with GC2 for golf shots.

**Comment by springbok** [comment]
I'm not familiar with the Foresight connection to GSPro but would it be possible to use the relay server? You could point the Foresight connector at the MLM2PRO connector's relay server, this would allow you to use ExPutt. Config would be: Foresight <-> relay server/exputt <-> gspro

**Reply by dmike2b** [reply]
I don't know how to use the relay server. But since I'm using the GSPro Simple Control Connector (and not the OpenAPI Connector) I am asuming that I can only use 1 connector. I wouldn't know how I would connect my GC2 to this relay server...

**Reply by springbok** [reply]
Ah ok yes the relay server only works if your connector uses the OpenAPI. I'd have to look into how the  GSPro Simple Control Connector works to see if we can integrate it.

**Reply by kl0tski** [reply]
Any updates on the data from the simple connector stream?

**Reply by gsgorveatt1234** [reply]
how does the relay server actually work. How do you point another connector to MLM2PRO connector.
